CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Unreleased
  - Fix: update to Gradle 7 [#432](https://github.com/logstash-plugins/logstash-input-beats/pull/432)
- - [DOC] Edit documentation for `executor_threads` [#tbd](https://github.com/logstash-plugins/logstash-input-beats/pull/tbd)
+ - [DOC] Edit documentation for `executor_threads` [#435](https://github.com/logstash-plugins/logstash-input-beats/pull/435)
 
```

Comment:
@andsel Would you like for me to bump the version and publish now?

Reply:
Yes @karenzone, thanks!